### PR TITLE
Support reference of document elements generated by LutaML other than Figures

### PR DESCRIPTION
**Pull request by w00lf**
https://github.com/metanorma/metanorma-plugin-lutaml/issues/41 new collection of macroses: `lutaml_table_package`, `lutaml_table_class`, `lutaml_table_enum`, `lutaml_table_data_type`, lookup xmi entities in the document by their name

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -382,6 +382,10 @@ In addition to the macros listed above that can be used only inside `lutaml_uml_
 [source,adoc]
 -----
 This is lutaml_figure::[package="Wrapper root package", name="Fig B1 Full model"] figure
+This is lutaml_table_package::[package="Wrapper root package"] package
+This is lutaml_table_class::[package="Wrapper root package", name="my name"] class
+This is lutaml_table_enum::[package="Wrapper root package", name="my name"] enum
+This is lutaml_table_data_type::[package="Wrapper root package", name="my name"] data_type
```
Comment (ronaldtse) [inline README.adoc:388]: Can we simplify these commands into:
* `lutaml_uml_table::[package="foo"]` 
* `lutaml_uml_table::[package="foo",class="bar"]`
* `lutaml_uml_table::[package="foo",enum="bar"]`
* `lutaml_uml_table::[package="foo",datatype="bar"]`
?

File: README.adoc [hunk 2]
```diff
@@ -418,10 +422,11 @@ Content for CityGML package
...
 ==== Defining tables
 
-.<<tab-P-another-C-{XMI_ID}>> -- Elements of &#8220;Another::AbstractAtomicTimeseries&#8221; (class)
+[arabic]
+.<<section-EAPK_9C96A88B_E98B_490b_8A9C_24AEDAC64293>> -- Elements of Another::AbstractAtomicTimeseries
 
-[[tab-P-another-C-{XMI_ID}]]
-.Elements of &#8220;Another::AbstractAtomicTimeseries&#8221; (class)
+[[section-EAPK_9C96A88B_E98B_490b_8A9C_24AEDAC64293]]
+.Elements of Another::AbstractAtomicTimeseries
```
Comment (ronaldtse) [inline README.adoc:429]: Add ` (class)` at the end of the caption title.

File: README.adoc [hunk 3]
```diff
@@ -418,10 +422,11 @@ Content for CityGML package
 
 ==== Defining tables
 
-.<<tab-P-another-C-{XMI_ID}>> -- Elements of &#8220;Another::AbstractAtomicTimeseries&#8221; (class)
+[arabic]
+.<<section-EAPK_9C96A88B_E98B_490b_8A9C_24AEDAC64293>> -- Elements of Another::AbstractAtomicTimeseries
```
Comment (ronaldtse) [inline README.adoc:426]: Remove `[arabic]` and add ` (class)` at the end of caption title.

Also, sometimes the package name (here as "Another") can contain spaces, so we should "double quote" the package name/class to differentiate 'Elements of "package name one::two"'.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (w00lf) [reply README.adoc:388]: yes, sure

**Pull request merged**